dali/ft/fttransform.cpp

```diff
@@ -642,9 +643,12 @@ void TransferServer::deserializeAction(MemoryBuffer & msg, unsigned action)
     ForEachItemIn(i1, progress)
         progress.item(i1).deserializeExtra(msg, 1);
 
+    if (msg.remaining())
+        msg.read(fileUmask);
```

Comment:
@AttilaVamos - do we know this is safe when remote system has older version of ftslave ?